components/channel_notifications_modal/channel_notifications_modal.scss

```diff
@@ -0,0 +1,139 @@
+.modal .channel-notifications-settings-modal {
+    overflow: hidden;
+    width: 100%;
+    max-width: 600px;
+    margin-top: -24px;
```

Comment:
we should be careful with negative margins, for the modal it should be ok but if we can avoid them i'd advocate for it.